cimetrics/env.py

```diff
@@ -68,6 +76,8 @@ def github_token(self) -> str:
 
 
 class GitEnv(Env):
+    _target_branch = None
```

Comment:
This is a static class variable, but I think you meant to use an instance variable (AFAIK they need to go into `__init__`), even though here it works by accident.

Reply:
This is deliberate, it's not an accident. Instance variables get initialised from static/class variable, it's a very common pattern to set attribute defaults in Python.

```
>>> class C:
...     a = 5
... 
>>> c1 = C()
>>> c1.a = 6
>>> c2 = C()
>>> c2.a
5
```

Reply:
See: https://rushter.com/blog/python-class-internals/

Reply:
Didn't know that, ok, all fine then.